pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -823,7 +835,10 @@ def expval(self, observable, shot_range=None, bin_size=None):
                             device_wires, qml.matrix(observable).ravel(order="C")
                         )
                 else:
-                    raise RuntimeError("LightningGPU-MPI does not currently support Hamiltonian.")
+                    device_wires = self.map_wires(observable.wires)
+                    return self._gpu_state.ExpectationValue(
+                        device_wires, qml.matrix(observable).ravel(order="C")
+                    )
 
             par = (
                 observable.parameters
```

Comment:
Do we add any safety checks for the adjoint pipeline to check if unsupported observables are passed in?